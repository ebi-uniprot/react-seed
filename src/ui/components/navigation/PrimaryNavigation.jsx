
import React, { Component } from 'react';

import Button from '../../elements/form/Button';

class PrimaryNavigation extends Component {
  state = {
    items: [
      'Home',
      'Downloads',
      'Contact',
    ],
  };

  render() {
    return (
      <div className="primary-navigation">
        { this.state.items.forEach(item => <Button>{item}</Button>) }
      </div>
    );
  }
}

export default PrimaryNavigation;
