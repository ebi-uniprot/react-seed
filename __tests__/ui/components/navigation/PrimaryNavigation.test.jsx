
import React from 'react';
import renderer from 'react-test-renderer';

import PrimaryNavigation from '../../../../src/ui/components/navigation/PrimaryNavigation';

describe('PrimaryNavigation component', () => {
  test('should render', () => {
    const component = renderer
      .create(<PrimaryNavigation />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
