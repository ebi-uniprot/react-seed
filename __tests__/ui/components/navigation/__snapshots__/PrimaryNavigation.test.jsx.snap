// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryNavigation component should render 1`] = `
<div
  className="primary-navigation"
/>
`;
